app/src/main/res/values-sr/strings.xml

```diff
@@ -0,0 +1,144 @@
+    <string name="preference_message_history_header">Istorija poruka</string>
+    <string name="preference_notification_header">Notifikacije</string>
+    <string name="whispers">Šaputanja</string>
+    <string name="reload_emotes">Osveži emotove</string>
+    <string name="preference_timestamp_title">Prikaži vremensku markicu</string>
+    <string name="preference_roomstate_title">Prikaži informacije o stanju sobe</string>
+    <string name="preference_roomstate_summary">Prikazuje informacije o trenutnom stanju sobe, poput režima samo za pretplatnike</string>
+    <string name="preference_streaminfo_title">Prikaži informacije o strimu</string>
+    <string name="preference_streaminfo_summary">Prikazuje da li je strim aktivan i broj gledalaca</string>
+    <string name="preference_notification_title">Prikaži obaveštenja o pominjanju</string>
+    <string name="preference_notification_summary">Stvara obaveštenja kada te neko pomene dok je aplikacija u pozadini</string>
+    <string name="preference_show_input_title">Prikaži unos</string>
+    <string name="preference_show_input_summary">Pirkazuje polje za unos za slanje poruka</string>
+    <string name="preference_show_timed_out_messages_title">Prikaži obrisane poruke</string>
+    <string name="preference_notifications_mentions_header">Notifikacije i pominjanja</string>
+    <string name="preference_timestamp_format_title">Format vremenskih markica</string>
+    <string name="snackbar_data_reloaded">Emotovi su osveženi</string>
+    <string name="preference_suggestions_title">Emotovi i korisnički predlozi</string>
+    <string name="preference_suggestions_summary">Prikaži predloge za emotove i aktivne korisnike dok kucate</string>
+    <string name="preference_tts_title">Omogući čitanje (tekst u govor)</string>
+    <string name="preference_tts_summary">Čita poruke aktivnog kanala</string>
+    <string name="preference_tts_queue_title">Režim rada poruka</string>
+    <string name="preference_tts_queue_add">Čita svaku poruku koristeći red</string>
+    <string name="preference_tts_queue_flush">Čita samo najnoviju poruku</string>
+    <string name="preference_tts_message_format_message">Čita samo poruku</string>
+    <string name="preference_tts_message_format_combined">Čita korisnika i poruku</string>
+    <string name="preference_tts_header">Tekst u govor</string>
+    <string name="preference_tts_message_format_title">Režim formata poruka</string>
+    <string name="preference_supibot_suggestions_title">Predlozi komandi za Supibot</string>
+    <string name="preference_scrollback_length_title">Dužina pomicanja poruka</string>
+    <string name="nuuls_upload_disclaimer">Spoljašni hosting pruža https://nuuls.com/i, korisite ga na sopstveni rizik</string>
+    <string name="nuuls_upload_title">Prilagođeno slanje slika</string>
+
+    <plurals name="viewers">
+        <item quantity="one">Uživo sa %d gledaocem</item>
+        <item quantity="other">Uživo sa %d gledaoca</item>
```

Comment:
Android Studio says the "few" quantity should be translated as well in Serbian, what do you think? https://i.nuuls.com/euGur.png

Reply:
Ok the android studio is right, but it is very hard to translate "few" in Serbian. Tried to look how other websites translated live count (like fb, and yt) but they only made an eye icon instead of the text, so I looked at  some local websites and rewrote it completely.  Now it says "Live ( %d people/person are/is watching)". 
Now it doesn't need "few". Quantity "other" should cover everything.
If you still want to have quantity "few" than I recommend to be exactly the same message as "one", but I am not 100% sure if it's the right translation because personally I would use different phrase for 10, 11 and 12+ for an example.

Reply:
Maybe you can use other quantity keywords as well? https://i.nuuls.com/5JNK6.png
If not I'll just mark the warning as invalid and merge it, just let me know 👍

Reply:
Sorry for taking so long to reply, yea just mark the warning as invalid, they way that I wrote non of those keywords will make a difference.